pkg/cli/config/config.go

```diff
@@ -0,0 +1,58 @@
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/openshift/backplane-cli/pkg/info"
+)
+
+type BackplaneConfiguration struct {
+	URL       string
+	Proxy_URL string
+}
+
+func GetBackplaneConfigFile() string {
+	userHomeDir, _ := os.UserHomeDir()
```

Comment:
Instead of storing the configuration in user's `$HOME`, `$HOME/.config` would be an ideal place to store config files I suppose?

```suggestion
	userConfigDir, _ := os.UserConfigDir()
```